include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -84,22 +84,23 @@ enum EOverlayType : byte {
 
 table CartesianImpedenceControlMode {
   /// actual stiffness to set rot:[nm/rad]
-  stiffness:EulerPoseParams;
-
+  stiffness:EulerPose;
```

Comment:
Is it really best to make these parameters required when they were previously optional? I believe reasonable defaults are set in ProcessData which is why this field is optional here.